content/docs/setup/kubernetes/multicluster-install/gateways/index.md

```diff
@@ -0,0 +1,242 @@
+---
+title: With Gateways
+description: Istio mesh spanning multiple Kubernetes clusters using Istio Gateway to reach remote pods.
+weight: 2
+keywords: [kubernetes,multicluster,federation,gateway]
+---
+
+Instructions for spanning an Istio mesh across multiple clusters when pods
+in each cluster can only talk to remote gateway IPs.
+
+## Prerequisites
+
+* Two or more Kubernetes clusters with **1.9 or newer**.
+
+* The ability to deploy the [Istio control plane](/docs/setup/kubernetes/quick-start/)
+on **each** Kubernetes cluster.
```

Comment:
Prerequisites should not be intangible things such as an "ability" or "understanding". Instead, use concrete, easily achievable, requisites:
* Read the instructions to deploy the [Istio control plane](/docs/setup/kubernetes/quick-start/)
on **each** Kubernetes cluster.
Remember to have the content indented on line 16 for consistency.